census/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "7f4f3919-5b0a-4b4b-93e5-7f0c035f3887",
+  "app_id": "census",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send your Census sync metrics and events to Datadog.",
+    "title": "Census",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Census Syncs Overview - Dashboard",
+        "image_url": "images/census_dashboard.png"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Category::Automation",
+      "Category::Monitoring",
+      "Offering::Integration",
+      "Submitted Data Type::Metrics",
+      "Submitted Data Type::Events"
+    ]
+  },
+  "assets": {
+    "dashboards": {
+      "Census Overview": "assets/dashboards/census_overview.json"
+    },
+    "integration": {
+      "source_type_name": "Census",
+      "configuration": {},
+      "events": {
+        "creates_events": true
+      },
+      "metrics": {
+        "prefix": "census",
```

Comment:
Will this conflict with internal metrics we already have defined?  We have an internal metadata store named [census](https://datadoghq.atlassian.net/wiki/spaces/DEC/pages/2508750859/Census#What-is-Census%3F) already and we're sending custom metrics prefixed with `census` so I don't want to cause confusion.